kernelci/config/email.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2022 Collabora Limited
+# Author: Alexandra Pereira <alexandra.pereira@collabora.com>
+#
+# This module is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation; either version 2.1 of the License, or (at your option)
+# any later version.
+#
+# This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+import yaml
+
+from kernelci.config.base import YAMLObject
+
+
+class Email(YAMLObject):
+    def __init__(self, email_type, email_send_to,
+                 email_send_from, email_subject):
+        self._email_type = email_type
+        self._email_send_to = email_send_to
+        self._email_send_from = email_send_from
+        self._email_subject = email_subject
+
+    @classmethod
+    def from_yaml(cls, email_type, kw):
+        return cls(email_type, **kw)
+
+    @property
+    def email_type(self):
+        return self._email_type
+
+    @property
+    def email_send_to(self):
+        return self._email_send_to
+
+    @property
+    def email_send_from(self):
+        return self._email_send_from
```

Comment:
I don't think the "from" field should be configurable in YAML, it's a system-wide parameter like the SMTP port etc.

Reply:
It makes sense. I removed subject as well.. at this moment not sure about what we should have in this YAML other than recipients list and the environment. Should I consider putting anything else?